notebook/images/ccg-workshop/Dockerfile

```diff
@@ -116,8 +116,14 @@ RUN wget -O WEScourse.ISB101.2018.tar.gz https://www.dropbox.com/s/fwqy9vze5p7w7
     tar -xf WEScourse.ISB101.2018.tar.gz && \
     rm -rf WEScourse.ISB101.2018.tar.gz
 
+RUN wget -O https://www.dropbox.com/s/80d3km03yhnhlh1/awk.tar?dl=1 && \
```

Comment:
does this actually work? I was surprised to learn that `-O` on debian's wget is documented thusly:
```
  -O,  --output-document=FILE      write documents to FILE
```

Reply:
oops, didn't test. fixed.